app/mqtt_listener.py

```
@@ -0,0 +1,96 @@
+        topic = msg.topic
+
+        try:
+            payload = msg.payload.decode("utf-8")
+            data = json.loads(payload)
+
+            if topic == self._settings.sensor_raw_topic:
+                try:
+                    point = sensor_raw_to_point(data)
+                    self._writer.write_point(point)
+                except (KeyError, ValueError, TypeError) as e:
+                    logger.error("Payload validation failed for %s: %s", topic, e)
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)